dev-utils/src/commands/init_command.rs

```diff
@@ -0,0 +1,225 @@
+use super::genesis_command::GenesisCommand;
+
+use failure::Error;
+
+use super::genesis_dev_config::GenesisDevConfig;
+use read_input::prelude::*;
+use serde_json::json;
+use serde_json::Value as JsonValue;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::Path;
+use std::process::Command;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
```

Comment:
We don't have to derive `StructOpt` for `InitCommand`. `StructOpt` should only be derived for types used for parsing command line arguments. As far as I am able to understand, `InitCommand` is just an intermediate data type and has nothing to do with command line arguments. Correct me if I'm wrong.

Reply:
agree, i'll remove it.

Reply:
fixed